app/validators/json_swagger_validator.rb

```diff
@@ -2,7 +2,12 @@ class JsonSwaggerValidator
   def initialize(schema_name, payload)
     @payload = payload
 
-    swagger_yaml = YAML.load_file(Rails.root.join("swagger/v6/swagger.yaml"))
+    swagger_yaml = if schema_name == "/v7/assessments"
+                     YAML.load_file(Rails.root.join("swagger/v7/swagger.yaml"))
+                   else
+                     YAML.load_file(Rails.root.join("swagger/v6/swagger.yaml"))
+                   end
+
```

Comment:
rather than this, maybe we should just pass the version into this component and let it work out the paths?